_includes/benefactors.html

```diff
@@ -0,0 +1,66 @@
+<div class="full-width-bg component">
+  <div class="grid-wrapper">
+      <div class="width-12-12 width-12-12-m">
+        <h2>Quarkus is a proud member of the Commonhaus Foundation</h2>
+      </div>
+      <div class="width-7-12 width-12-12-m">
+        <p>The Commonhaus Foundation which is a non-profit dedicated to the enduring viability of open-source projects. It offers a neutral environment, a governing structure, and assistance to projects, prioritizing their independence and development led by the community. They furnish resources such as legal aid, infrastructure, and financial avenues, all while honoring each project's established governance and culture.</p>
+        <p><a href="https://www.commonhaus.org/" target="blank">Learn more about the Commonhaus Foundation</a> and how you can make a difference.</p>
+      </div>
+      <div class="width-5-12 width-12-12-m">
+        <a href="https://www.commonhaus.org/" target="blank">
+        <img class="light-only" src="https://raw.githubusercontent.com/commonhaus/artwork/main/foundation/brand/svg/CF_logo_horizontal_stack_default.svg" alt="Learn about the Commonhaus Foundation">
+        <img class="dark-only" src="https://raw.githubusercontent.com/commonhaus/artwork/main/foundation/brand/svg/CF_logo_horizontal_stack_reverse.svg" alt="Learn about the Commonhaus Foundation">
+        </a>
+      </div>
+    </div>
+  </div>
+
+   <div class="component-wrapper-slim">
+    <div class="grid-wrapper">
+      <div class="width-4-12 width-12-12-m">
+        <h3>IBM</h3>
+        <p>IBM strengthened its Java and Quarkus commitment by moving the Red Hat engineers working on Quarkus into IBM. This ensured dedicated resources for Quarkus's growth, deepening IBM's open-source involvement and focus on cloud-native Java for enterprises.</p>
+        <p><a href="https://www.ibm.com/opensource/" target="blank">Learn more about open source at IBM</a></p>
+      </div>
+      <div class="width-4-12 width-12-12-m">
+        <h3>Red Hat</h3>
+        <p>Quarkus originated at Red Hat, which ensured its ongoing development and stability by employing many of its principal engineers. This support  helped cultivate a strong ecosystem for cloud-native application development. A key reason for Quarkus's transition to the Commonhaus Foundation was Red Hat's dedication to its open-source nature.</p>
+        <p><a href="https://www.redhat.com/en/blog/what-does-open-source-program-office-do" target="blank">Learn more about open source at Red Hat</a></p>
+      </div>
+      <div class="width-4-12 width-12-12-m">
+        <h3>AWS</h3>
+        <p>AWS significantly aids Quarkus by donating credits, enabling developers to use AWS services for testing, CI, and feature development. This support accelerates Quarkus innovation, enhancing the cloud-native experience on AWS.</p>
```

Comment:
```suggestion
        <p>AWS aids Quarkus by donating credits, enabling developers to use AWS services for testing, CI, and feature development. This support accelerates Quarkus innovation, enhancing the cloud-native experience on AWS.</p>
```
I tend to prefer avoiding superlatives unless there's a very strong amazing reason.